src/util/domains/remove-domain-by-name.ts

```diff
@@ -16,11 +16,21 @@ export default async function removeDomainByName(
       return new ERRORS.DomainPermissionDenied(domain, contextName);
     }
     if (error.code === 'domain_removal_conflict') {
-      const { aliases, certs, suffix, transferring } = error;
+      const {
+        aliases,
+        certs,
+        message,
+        pendingAsyncPurchase,
+        resolvable,
+        suffix,
+        transferring
+      } = error;
       return new ERRORS.DomainRemovalConflict({
```

Comment:
I would make this half of the lines by accessing `error.<field>` when creating the error instance